index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="./style.css">
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css">
+
+    <style>
+        /* Stackoverflow preview fix, please ignore */
+        
+        .navbar-nav {
+            flex-direction: row;
+        }
+        
+        .nav-link {
+            padding-right: .5rem !important;
+            padding-left: .5rem !important;
+        }
+    </style>
+
+</head>
+
+<body>
+
+    <div class="container">
+        <header id="header">
+
+
+            <nav class="navbar navbar-expand-lg navbar-light bg-light" id="nav-bar">
+                <a class="navbar-brand" href="#"><img src="" alt="" id="header-img">Logo</a>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                  <span class="navbar-toggler-icon"></span>
+                </button>
+
+
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item">
+                        <a class="nav-link" href="#features">Features</a></a>
+                    </li>
+                    <li class="navbar-item">
+                        <a class="nav-link" href="#howitworks">How It Works</a>
+                    </li>
+                    <li class="navbar-item">
+                        <a class="nav-link" href="#pricing">Pricing</a>
+                    </li>
+                </ul>
+
+            </nav>
+
+
+        </header>
+
+        <div class="container text-center">
+            <section class="text-center">
+                <h3 class="text-center">Handcrafted, home-made masterpieces</h3>
+
+                <form id="form" action="https://www.freecodecamp.com/email-submit">
+                    <input name="email" id="email" type="email" placeholder="Enter your email address" required /> <br>
+                    <input id="submit" type="submit" value="Get Started" class="btn btn-warning mt-3" />
+                </form>
+
+            </section>
+
+            <section class="text-center mt-5" id="features">
+                <h3>Premium Materials</h3>
+                <div class="row mt-2">
+                    <div class="col-2">
+
+                    </div>
+                    <div class="col-10">
+
+                        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+                    </div>
```

Comment:
- This Paragraph text should be properly aligned to the center.

Reply:
- [ ]  Please always try to sound polite. Avoid sentences that come off as commands. You should also be clear in your suggestions. You can also use screen shots to show what you are requesting to be changed.